### ISSUE TITLE
unknown option: -target even with example from github 

### ISSUE THREAD

**Issue by newbieflutter** [OneSignal/OneSignal-Flutter-SDK#260]
Hi I have an application where I am trying to integrate one signal all works fine but when I tried to run this command flutter build ios. I get this error

```
Xcode's output:
↳
    ld: unknown option: -target
    ld: unknown option: -target
    note: Using new build system
    note: Building targets in parallel
    note: Planning build
    note: Constructing build description
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 4.3, but the range of supported deployment target versions is 8.0 to 13.5.99. (in target 'OneSignal' from project 'Pods')
```

For you information I have downloaded the example from github assign my provision details etc. Also the same error when I tried to build the example application? What is best resolution to this?

**Comment by rgomezp** [comment]
Duplicate of https://github.com/flutter/flutter/issues/59974

**Comment by rgomezp** [comment]
Closing since this isn't a OneSignal specific issue. Cheers
